frontend/packages/kubevirt-plugin/integration-tests/tests/models/uiResource.ts

```diff
@@ -1,25 +1,29 @@
+import { isEmpty } from 'lodash';
 import { K8sKind } from '@console/internal/module/k8s/types';
 import { resourceTitle } from '@console/internal-integration-tests/views/crud.view';
 import { click } from '@console/shared/src/test-utils/utils';
 
 import { getResourceObject } from '../utils/utils';
 import * as view from '../../views/uiResource.view';
 
+const apiVersionForModel = (model: K8sKind) =>
```

Comment:
@suomiy I wasn't able to import this from @console/internal/model (circular dependency issue?)

Reply:
have you tried long import like this?
`import { apiVersionForModel } from '@console/internal/module/k8s/k8s';`

in case that doesn't work, can you move this to some utils file and write a short comment why it is duplicated